### DISCUSSION TITLE
Getting django-stubs working with pytest-mypy-plugins

### DISCUSSION THREAD

**Discussion by delfick** [typeddjango/django-stubs#2031] [Q&A] [answered]
Hi,

I'm trying to make the ability to deal with https://github.com/typeddjango/django-stubs/issues/1684 in a way that doesn't cause other problems. I technically have a proof of concept, but I want to work on it a lot more before I bring it up as any kind of solution.

My problem at the moment though is I want to be able to run this code in our existing project before we update django-stubs (we're on an old version of django-stubs/mypy in a mammoth codebase and upgrading is difficult) and so I'm developing it here https://github.com/delfick/extended-mypy-django-plugin as it's own separate thing that uses django-stubs as a base while I get it to not be awful.

My problem today is I'm trying to make it use pytest-mypy-plugins but I get this output 

```
E     ../../../../../../Users/runner/work/extended-mypy-django-plugin/extended-mypy-django-plugin/tools/.python/lib/python3.10/site-packages/django-stubs/db/models/lookups.pyi:4: error: Cannot find implementation or library stub for module named "db.models"  [import-not-found] (diff)
E     ../../../../../../Users/runner/work/extended-mypy-django-plugin/extended-mypy-django-plugin/tools/.python/lib/python3.10/site-packages/django-stubs/db/models/lookups.pyi:4: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports (diff)
E     ../../../../../../Users/runner/work/extended-mypy-django-plugin/extended-mypy-django-plugin/tools/.python/lib/python3.10/site-packages/django-stubs/db/models/sql/where.pyi:4: error: Cannot find implementation or library stub for module named "db.models"  [import-not-found] (diff)
```

https://github.com/delfick/extended-mypy-django-plugin/actions/runs/8445039553/job/23131609062

I'm wondering if there's anything obvious that I'm not doing in that repository that will make it work, or if there's some reason that it can't be made to work?

Thanks.

**Comment by sobolevn** [comment]
This works for us: https://github.com/typeddjango/django-stubs/blob/3f9b3c82c3038663beb49fb41c05734eedd95e8b/.github/workflows/test.yml#L64-L71

**Reply by delfick** [reply, marked as answer]
I had played with this to no effect.

The good news is a break from the computer has given me relatively fresh eyes and the problem is actually that your mypy.ini has `ignore_missing_imports = true` and mine did not. Adding that makes those go away. thanks :)
